currency/rate_converter_test.go

```diff
@@ -169,7 +168,6 @@ func TestRateStaleness(t *testing.T) {
 	defer mockedHttpServer.Close()
 
 	expectedRates := &Rates{
-		DataAsOf: time.Date(2018, time.September, 12, 0, 0, 0, 0, time.UTC),
```

Comment:
Can we simplify 
`initialFakeTime := time.Date(2018, time.September, 12, 30, 0, 0, 0, time.UTC)`  L181 (and other tests) to `initialFakeTime := time.Now()` ? 
I'm ok if you think it doesn't make much sense. 
Other than this this PR looks good to me.

Reply:
It won't be necessary because this PR removes the `Rates` struct's `DataAsOf    time.Time` field altogether.